docker-compose.yml

```diff
@@ -130,6 +130,15 @@ services:
       - "9200:9200"
       - "9300:9300"
 
+  # https://www.docker.elastic.co/
+  elasticsearch7:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.1.0
```

Comment:
They have two flavors of images--one that's just open source software and one that has a bunch of additional proprietary stuff. I'm pretty sure we're just going to use the oss one in stage/prod, so we should use that in the local dev environment.